assets/js/editor-components/sidebar-compatibility-notice/index.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import { Notice, ExternalLink } from '@wordpress/components';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { useCompatibilityNotice } from '@woocommerce/editor-components/compatibility-notices/use-compatibility-notice';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+export const CartCheckoutSidebarCompatibilityNotice = ( {
+	block,
+}: {
+	block: 'cart' | 'checkout';
+} ) => {
+	const [ isVisible, dismissNotice ] = useCompatibilityNotice( block );
+
+	const noticeText = createInterpolateElement(
+		__(
+			'The Cart & Checkout Blocks are a beta feature to optimize for faster checkout. To make sure this feature is right for your store, <a>review the list of compatible extensions</a>.',
+			'woo-gutenberg-products-block'
+		),
+		{
+			a: (
+				// Suppress the warning as this <a> will be interpolated into the string with content.
+				// eslint-disable-next-line jsx-a11y/anchor-has-content
+				<ExternalLink href="https://woocommerce.com/document/cart-checkout-blocks-support-status/#section-7" />
```

Comment:
We should redirect to the list of `Compatible Extensions` (i.e., this link instead: https://woocommerce.com/document/cart-checkout-blocks-support-status/#section-3). In the issue description that @ralucaStan wrote #6851, we are mentioning the list of `Incompatible Extensions` instead. But, on the Figma file and on the context post pdFofs-b7-p2, we want to redirect to the `Compatible Extensions`.